### PR TITLE
Renames Expense's value to amount.

### PR DESCRIPTION
**Pull request by bsonntag**
With this the field is now consistent with PeriodicExpense's amount.

### CODE REVIEW

**Comment by bsonntag** [comment]
@zamith bump

File: spec/models/expense_spec.rb [hunk 1]
```diff
@@ -16,8 +16,8 @@
       expect(expense).to be_valid
     end
 
-    it 'must have a value' do
-      expense = build(:expense, value: nil)
+    it 'must have an amount' do
+      expense = build(:expense, amount: nil)
 
       expect(expense).not_to be_valid
```
Comment (zamith) [inline spec/models/expense_spec.rb]: This could be `expect(expense).to be_invalid`, which I think is nicer, since it is phrased in the positive.

Comment (bsonntag) [inline spec/models/expense_spec.rb]: I also prefer positive statements, I just didn't know I could do that. Thanks for pointing that out.

Comment (bsonntag) [inline spec/models/expense_spec.rb]: Done.

**Comment by zamith** [comment]
:+1:

**Pull request merged**